Formula/haxe.rb

```diff
@@ -109,7 +111,13 @@ class HelloWorld {
       }
     EOS
     system "#{bin}/haxe", "-js", "out.js", "-main", "HelloWorld"
-    _, stderr, = Open3.capture3("osascript -so -lJavaScript out.js")
-    assert_match "Hello world!", stderr.strip
+    output = nil
+    on_macos do
+      _, output, = Open3.capture3("osascript -so -lJavaScript out.js")
```

Comment:
This can be `shell_output` too I think

Reply:
Should be possible if we redirect stderr (`2>&1`). Not sure why it outputs to stderr though. I tried changing `-s` value but didn't do anything.